python/pyspark/sql/window.py

```diff
@@ -76,12 +79,37 @@ def partitionBy(*cols):
 
     @staticmethod
     @since(1.4)
-    def orderBy(*cols):
+    def orderBy(*cols, **kwargs):
         """
         Creates a :class:`WindowSpec` with the ordering defined.
+
+        :param cols: names of columns or expressions.
```

Comment:
We should add ` ... versionchanged:: 2.5 blah blah` BTW

Reply:
Hi @HyukjinKwon , I've added version directive. Thanks